x-pack/plugins/index_lifecycle_management/public/sections/edit_policy/components/node_attrs_details/node_attrs_details.js

```diff
@@ -15,10 +17,8 @@ import {
   EuiSpacer,
   EuiPortal,
 } from '@elastic/eui';
-import { FormattedMessage, injectI18n } from '@kbn/i18n/react';
 
-
-export class NodeAttrsDetailsUi extends PureComponent {
+export class NodeAttrsDetails extends PureComponent {
```

Comment:
TODO: Verify this looks correct in the UI.

Reply:
![image](https://user-images.githubusercontent.com/1238659/59351540-7e82cc00-8cd3-11e9-9072-dc8da3efcfcd.png)